lib/pbench/test/functional/agent/cli/commands/conftest.py

```diff
@@ -23,9 +23,9 @@ def pbench_cfg(tmp_path, opt_pbench):
 
 @pytest.fixture
 def pbench_run(monkeypatch, tmp_path):
+    monkeypatch.delenv("pbench_run")
```

Comment:
Sure; that makes sense. We're creating a new `var/lib/pbench-agent` under some transient temporary, and I like the idea of making sure there's no existing `pbench_run` environment variable pointing somewhere else.